src/Components/Dashboard/Feeds.js

```diff
@@ -1,110 +1,100 @@
-import React from 'react';
+import React, { useState } from 'react';
 import Like from '../Assets/like.png';
 import isLike from '../Assets/liked.png';
 import moment from 'moment';
 import commentIcon from '../Assets/comment.png';
-import LoadMore from './LoadMore';
+import { EmojiEmotionsOutlined } from '@material-ui/icons';
 import { getBackgroundColor } from '../../Utils/colors';
-import { ColorRing } from 'react-loader-spinner';
+import { ColorRing, Oval } from 'react-loader-spinner';
 import Unavailiabe from '../../Utils/unavailiable1.png';
+import InfiniteScroll from 'react-infinite-scroll-component';
+import { getFeedDetails } from '../../Auth/Actions/feedActions';
+import { useDispatch } from 'react-redux';
+
+const Feeds = ({
+    feeds,
+    setData,
+    setPostArticle,
+    setPostArticleModal,
+    offSet,
+    setOffSet,
+    feedsLength,
+    requesting,
+}) => {
+    let pageYOffset = window.pageYOffset;
+    // console.log(feeds);
+    const dispatch = useDispatch();
+    const [hasMore, setHasMore] = useState(true);
+    const fetchMoreData = () => {
+        if (feedsLength >= 10) {
+            setTimeout(() => {
+                setOffSet(offSet + 10);
+                const newOffset = offSet + 10;
+                dispatch(getFeedDetails(newOffset)).then((res) => {
+                    pageYOffset = window.pageYOffset;
+                    setData(feeds.concat(res.data.data));
+                });
+            }, 3000);
+        } else {
+            setHasMore(false);
+        }
+    };
+    // console.log(pageYOffset);
+    React.useLayoutEffect(() => {
```

Comment:
why do you need to scroll?

Reply:
i used it to check if i can pageOffset to the current scroll position due to the issues with loading more data with a mobile phone